packages/system/src/system.utils.ts

```diff
@@ -35,6 +35,7 @@ export const domElements = [
   'h6',
```

Comment:
Nice work here. Please remember to add a changeset with a minor bump for all changes made to the core components.

Reply:
Hi, Thanks for the feedback!

So if I'm right, I should run `yarn changeset` and then include all changed packages in a minor bump? 😄